pkg/kubectl/resource_printer.go

```diff
@@ -600,15 +641,6 @@ func printPodBase(pod *api.Pod, w io.Writer, withNamespace bool, wide bool, show
 	return err
 }
 
-func printPodList(podList *api.PodList, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
```

Comment:
Are these two functions moved without code changes? It might be better to keep them where they were, and make it easier to keep track of code changes.